code/game/machinery/computer/atmos_control.dm

```diff
@@ -289,6 +290,12 @@ GLOBAL_LIST_EMPTY(atmos_air_controllers)
 		if("input")
 			signal.data += list("tag" = input_tag, "power_toggle" = TRUE)
 			. = TRUE
+		if("rate")
+			var/target = input("New target rate:", name, input_info ? input_info["volume_rate"] : 0) as num|null
+			if(!isnull(target) && !..())

```

Comment:
Why are you calling ..() here?

Reply:
copypaste https://github.com/tgstation/tgstation/pull/41674/files/1e6ea0ffb6b19f03a1b6574a04a5ee706861d226#diff-20915160298ed00ab622c574f8f1c5a0R304